COMET.Web.Common/Server/Services/ConfigurationService/ConfigurationService.cs

```diff
@@ -64,13 +73,22 @@ public override Task InitializeService()
                 return Task.CompletedTask;
             }
 
+            this.ServerConfiguration = new ServerConfiguration();
```

Comment:
A small improvement that could be done there (I just think about that now) : Could it be possible to have a complete ServerConfiguration inside the app.settings.json file? We could then use only one JsonSerializer call instead of querying each properties of the IConfiguration ?

Reply:
This is a great suggestion. I've done that on my latest commit - for this to work, I had to modify our server_configuration.json a bit, so that we could have a ServerConfiguration object. This way I could just get the ServerConfiguration section, which then retrieves all of the properties within that object.
I had to do it this way because IConfiguration doesn't offer an easy way for us to just read the whole file - only section by section. Should work as what we want 😉